package.json

```diff
@@ -74,5 +74,6 @@
         "hooks": {
             "pre-commit": "pretty-quick --staged"
         }
-    }
+    },
+    "packageManager": "yarn@1.22.21+sha1.1959a18351b811cdeedbd484a8f86c3cc3bbaf72"
```

Comment:
Why add this configuration? 🤔

Reply:
`corepack`  can't live without it unless you set env variable  `COREPACK_ENABLE_AUTO_PIN` to `0`, which is quite inconvinient to do. But now when thinking about it, `yarn` maybe also checks this fields, so it may cause issues to non-corepack users, right?

Reply:
Yea, there is a [`SKIP_YARN_COREPACK_CHECK` env var for `yarn`](https://github.com/yarnpkg/yarn/releases/tag/v1.22.21), but this is equally bad as the need to set `COREPACK_ENABLE_AUTO_PIN`. Feel free to revert the change if it's causing problems, unless you know of a better solution of course.

Reply:
https://github.com/yarnpkg/yarn/issues/9015